pkg/kubeconf/clientset.go

```diff
@@ -30,6 +26,26 @@ func createClientset() *kubernetes.Clientset {
 	return clientset
 }
 
+// get a valid kubeconfig path
+func getKubeConfig() string {
+	var kPath string
+	if os.Getenv("KUBECONFIG") != "" {
+		kPath = os.Getenv("KUBECONFIG")
+	} else if home := os.Getenv("HOME"); home != "" {
```

Comment:
do we want to support windows? if so we would need to check `os.Getenv("USERPROFILE")` in addition to `os.Getenv("HOME")`.  I don't personally feel strongly either way.

Reply:
good idea - i hadn't even thought about windows!  I'll work on getting that added

Reply:
yeah i normally try to block windows out of my mind. lol.

Reply:
oh actually, check this out, in the [root.go file](https://github.com/JessicaGreben/health-check/blob/master/cmd/root.go#L7) it imports a library to do the `home` evaluations.  That might be nice to use here instead since we are already using it.  

You can see the implementation of that library here:
https://github.com/mitchellh/go-homedir/blob/master/homedir.go